SholaQ.html

```diff
@@ -11,7 +11,7 @@
 <body>
     <div class="header">
         <div class="navbar">
-            <a href="#contact">Contact</a>
```

Comment:
Revisit all of your anchor links to ensure that they are all working as expected.